vite.config.ts

```diff
@@ -3,6 +3,7 @@ import { defineConfig } from "vitest/config";
 export default defineConfig({
   test: {
     setupFiles: ["./setupFile.ts"],
+    globalSetup: ["./tests/helpers/globalSetup.ts"],
```

Comment:
Leave some comments here explaining how globalSetup is working.

Reply:
Done.